<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>572</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Tool for Grade Calucation</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="default_tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>(you first need to create classes, using the Add Class button to the right. If you don't find your previously created classes, try &quot;Change Location of DB&quot; and select the folder where you have stored the exams the last time).</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="remove_tab">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QTreeView" name="treeView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="button_details">
              <property name="text">
               <string>Details</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_addClass">
              <property name="text">
               <string>Add Class</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_deleteClass">
              <property name="text">
               <string>Delete Class</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="button_saveDB">
              <property name="text">
               <string>Save Database</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_changeLocation">
              <property name="text">
               <string>Change Location of DB</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_openDB">
              <property name="text">
               <string>Open Database</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="text">
         <string>Some text can come here, like a description of the program or so. I'll have to figure out line breaks or text wrapping, because this way it looks shitty.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Databases">
     <property name="title">
      <string>Recent Databases</string>
     </property>
     <addaction name="actionOption_1"/>
    </widget>
    <addaction name="action_newDB"/>
    <addaction name="action_openDB"/>
    <addaction name="menuRecent_Databases"/>
    <addaction name="action_changeLocation"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_restoreData"/>
    <addaction name="action_cleanTrash"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_openTutorial"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_newDB">
   <property name="text">
    <string>New Database</string>
   </property>
   <property name="toolTip">
    <string>Create New Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_openDB">
   <property name="text">
    <string>Open Database</string>
   </property>
   <property name="toolTip">
    <string>Open Existing Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOption_1">
   <property name="text">
    <string>Option 1</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save on Disk</string>
   </property>
   <property name="toolTip">
    <string>Save DB on Disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_changeLocation">
   <property name="text">
    <string>Change Location</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_openTutorial">
   <property name="text">
    <string>Open Tutorial</string>
   </property>
  </action>
  <action name="action_restoreData">
   <property name="text">
    <string>Restore Data</string>
   </property>
  </action>
  <action name="action_cleanTrash">
   <property name="text">
    <string>Clean Trash</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
